src/main/java/com/google/cloud/tools/appengine/api/deploy/AppYamlProjectStaging.java

```diff
@@ -49,12 +45,14 @@
       ImmutableList.of("cron.yaml", "dos.yaml", "dispatch.yaml", "index.yaml", "queue.yaml");
 
   /**
-   * Stages a Java JAR/WAR App Engine application to be deployed.
+   * Stages an appengine app.yaml based project for deployment. Copies app.yaml, the project
+   * artifact and any user defined extra files. Will also copy the Docker directory for flex
```

Comment:
Should we take out the "flex" reference?

Reply:
the behavior is slightly different for flex unfortunately.